specreduce/utils/synth_data.py

```diff
@@ -7,6 +7,15 @@
 import astropy.units as u
 from astropy.modeling import models
 from astropy.nddata import CCDData
+from astropy.wcs import WCS
+from astropy.stats import gaussian_fwhm_to_sigma
+
+from specreduce.calibration_data import load_pypeit_calibration_lines
+
+__all__ = [
+    'make_2dspec_image',
+    'make_2d_arc_image'
```

Comment:
how about renaming `make_2dspec_image` to `make_2d_trace_image`? that's a more accurate description of what it does. this would open up the possibility of a `make_2d_spec_image` that combines `make_2d_trace_image` and `make_2d_arc_image`. might be worth implementing that as part of this PR.